src/components/screen-header/LargeScreenHeader.tsx

```diff
@@ -22,7 +22,15 @@ export type LargeScreenHeaderProps = {
   containerStyle?: ViewStyle;
   color?: StaticColor;
   setFocusOnLoad?: boolean;
+  buttonProps?: Omit<LargeHeaderButtonProps, 'color'>;
+  /**
+   * JSX content that will be displayed as part of the disappearing header,
+   * below the title, and above global messages.
+   */
   headerChildren?: JSX.Element | JSX.Element[];
+  /**
+   * Page content, below disappearing header.
+   */
```

Comment:
Is this all the remaining screen content? So everything shown in the screen should be children inside the LargeScreenHeader?

Reply:
Yes 👍